### PR TITLE
Included relevant detail for me

### PR DESCRIPTION
**Pull request by ciaranmckenna**

### CODE REVIEW

**Comment by ciaranmckenna** [comment]
Would like to be able to use the link for the MSc in a similar link to the St Mary's link

**Review by barrymcgee — CHANGES_REQUESTED** [review]
Do not add images (or any files) to the repo that are not being used anywhere. In this case, remove the `.svg` files if you aren't using them.

File: src/pages/resume.js [hunk 1]
```diff
@@ -45,270 +48,145 @@ const Resume = () => (
...
-          </p>
-          <p>
-            Canonical&#039;s core mission to make open source software available
-            to people everywhere and one of the many perks of working with them
-            is the ability to contribute back to the open source community. One
-            highlight I'm currently working on is Vanilla Framework, a SASS
-            based framework of patterns to support all of Canonical&#039;s
-            upcoming web releases.
+          In my role as an Application Developer with Citi, 
+          I work with a development team to deliver continuous stability and enhancement to the e – Trading platform.
```
Comment (barrymcgee) [inline src/pages/resume.js]:
```suggestion
          I work with a development team to deliver continuous stability and enhancement to the e-Trading platform. 
```

I've removed the spaces around the hyphen here, does the official name internally definitely have a hyphen?

File: src/pages/resume.js [hunk 2]
```diff
@@ -45,270 +48,145 @@ const Resume = () => (
...
           <p>
-            The lab was a large corporate with multiple competing sub-brands so
-            the challenge was to develop an over-arching set of patterns &amp;
-            living components that could be thread through multiple products to
-            build a cohesive overall brand.
+          I worked as part of a development team to build a web-based asset management application. 
+          This is a front office application for boutique asset managers. 
+          The application compromises of compliance modelling, order generation and reconciliation modules.
+          The user interface was developed with Angular 4, for which I have received training. 
+          The styling of the application makes use of the Bootstrap, enabling a consistent and reactive design.
```
Comment (barrymcgee) [inline src/pages/resume.js]:
```suggestion
          The styling of the application makes use of Bootstrap, enabling a consistent and responsive design.
```

File: src/pages/resume.js [hunk 3]
```diff
@@ -45,270 +48,145 @@ const Resume = () => (
...
           </p>
         </div>
       </section>
       <section className="entry-list__item row">
         <div className="column">
           <span className="entry-list__meta">
-            <time datetime="2015-02-04">Feb</time> &ndash;
-            <time datetime="2015-10-16">Oct 2015 (Medium-term contract)</time>
+            <time datetime="2015-02-04">June</time> &ndash;
```
Comment (barrymcgee) [inline src/pages/resume.js]: You need to make sure the `datetime` attributes match the text values you place inside the tags.

File: src/pages/resume.js [hunk 4]
```diff
@@ -45,270 +48,145 @@ const Resume = () => (
...
           <p>
-            <em>Kahoot!</em> is a real-time quiz collaboration platform
-            experiencing huge growth - as of May '15, Kahoot has now been used
-            by over 50 million users worldwide.
-          </p>
-          <p>
-            I worked with Kahoot over a eight month period to provide guidance
-            and direction on improving their front end architecture for
-            scalability and modularity in the future. This involved:
+            <em>Aepona</em> are a software provider that offers secure enterprise
```
Comment (barrymcgee) [inline src/pages/resume.js]:
```suggestion
            <em>Aepona</em> are a software provider that offer secure enterprise
```

File: src/pages/resume.js [hunk 5]
```diff
@@ -45,270 +48,145 @@ const Resume = () => (
...
-            architecture which the maturing team can now inherit and build upon
-            further.
+          <a href="https://www.qub.ac.uk/courses/postgraduate-taught/software-development-msc/"></a>
+          During the academic year of 2016/17 I enrolled in the MSc Software Development
+          course offered by Queen's University Belfast.
+
+          This course gave me an opportunity to analyse and solve problems 
+          through the design of robust and elegant software systems. 
+          Skills developed included Java programming, web development, 
+          database design, and software testing.
```
Comment (barrymcgee) [inline src/pages/resume.js:137]: You should add a link to your blog post at the end of this section.

File: src/pages/resume.js [hunk 6]
```diff
@@ -45,270 +48,145 @@ const Resume = () => (
...
-            <img src={othermediaLogo} alt="Other Media logo" />
+            <img src={educationAutorityLogo} alt="Education Authority Logo" />
           </h3>
           <p>
-            In my time with <em>Other Media</em> I was leading a small Front End
-            team helping them transition to a more agile way of working (Scrum)
-            to deliver high profile responsive projects for large enterprise
-            clients such as Paul Smith and smaller high impact projects for
-            highly visible clients such as Jasper Conran.
+          I was nominated to enrole in the Professional Headship Qualification cohort of
```
Comment (barrymcgee) [inline src/pages/resume.js]:
```suggestion
          I was nominated to enroll in the Professional Headship Qualification cohort of 
```

File: src/pages/resume.js [hunk 7]
```diff
@@ -45,270 +48,145 @@ const Resume = () => (
...
           </h3>
           <p>
-            In my time with <em>Other Media</em> I was leading a small Front End
-            team helping them transition to a more agile way of working (Scrum)
-            to deliver high profile responsive projects for large enterprise
-            clients such as Paul Smith and smaller high impact projects for
-            highly visible clients such as Jasper Conran.
+          I was nominated to enrole in the Professional Headship Qualification cohort of 
+          2015/16, an accredited course for those aspiring to headship. 
+          This course provides opportunity for:
```
Comment (barrymcgee) [inline src/pages/resume.js]:
```suggestion
          This course provided opportunity for:
```

File: src/pages/resume.js [hunk 8]
```diff
@@ -45,270 +48,145 @@ const Resume = () => (
...
-            I worked at the <em>BBC</em> for just under 5 years in total,
-            progressing from a Junior Client Side Developer in 2007 at BBC
-            Switch to spending my last eighteen months on the BBC iPlayer team.
-          </p>
-          <p>
-            In this role, I worked within a large project team to deliver
-            continuous improvements and features to the BBC's high profile TV
-            &amp; iPlayer website.
+          I have worked in various capacities including being a member of the school senior management team, 
+          head of numeracy, head of PE &amp; Sport, Public Relations Officer,
```
Comment (barrymcgee) [inline src/pages/resume.js:176]:
```suggestion
          Head of Numeracy, Head of PE &amp; Sport, Public Relations Officer, 
```

**Review by barrymcgee — CHANGES_REQUESTED** [review]
This is excellent work @ciaranmckenna, CV section looks great! I've left a few comments and suggestions, work through them and ping me back for review.

**Comment by barrymcgee** [comment]
@ciaranmckenna Also, don't forget to remove the image imports you're not using.

screenshot 2019-02-23 at 17 50 29

File: src/pages/resume.js (hunk 1 above)
Comment (ciaranmckenna) [reply src/pages/resume.js]: I left the .svg in there as i thought you were going to do some magical work with them to resolve the image issue i was receiving

File: src/pages/resume.js (hunk 5 above)
Comment (ciaranmckenna) [reply src/pages/resume.js:137]: Fair point ... I was trying to do this at the start ... but unfortunately, I was unsuccessful ...

**Review by barrymcgee — APPROVED** [review]

**Pull request merged**